scripts/func_parser.py

```diff
@@ -149,9 +149,11 @@ def strip_line(l):
     """Delete all tabs"""
     return re.sub(' +',' ', l3)
 
-def create_func_db(filename):
-    with open(filename, 'r') as f:
-        data = f.readlines()
+def create_func_db(filenames):
+    data=[]
+    for filename in filenames.split(":"):
+        with open(filename, 'r') as f:
+            data.extend(f.readlines())
```

Comment:
This could use some explanations, why is this change needed?

Reply:
I tried fixing the scripts for adding a new backend, but at the end gave up and did everything manually. Reverting this change as it was not intended to be included in the first place.